build-tools/dependencies/dependencies.projitems

```diff
@@ -7,51 +7,61 @@
   <ItemGroup>
     <RequiredProgram Include="$(HostCcName)"        Condition=" '$(HostOS)' != 'Windows' " />
     <RequiredProgram Include="$(HostCxxName)"       Condition=" '$(HostOS)' != 'Windows' " />
-    <RequiredProgram Include="7za"                  Condition=" '$(NeedMxe)' == 'true' And $(AndroidSupportedHostJitAbisForConditionalChecks.Contains (':mxe-Win64:'))">
+    <RequiredProgram Include="x86_64-w64-mingw32-g++" Condition=" '$(HostOS)' == 'Darwin' " >
+      <Homebrew>mingw-w64</Homebrew>
+    </RequiredProgram>
+    <RequiredProgram Include="7za"                  Condition=" '$(HostOS)' == 'Darwin'">
       <Homebrew>p7zip</Homebrew>
     </RequiredProgram>
-    <RequiredProgram Include="autoconf"             Condition=" '$(HostOS)' != 'Windows' ">
+    <RequiredProgram Include="autoconf"             Condition=" '$(HostOS)' == 'Darwin' ">
```

Comment:
Does the Linux+mono build still require `autoconf` and company?  Shouldn't this still check for `!= 'Windows'`?

Reply:
Mono uses `autoconf` but Linux doesn't use this mechanism to ensure the package presence, it has the dependency scripts [in here](https://github.com/xamarin/xamarin-android/tree/master/build-tools/scripts/dependencies) so the above is relevant only for macOS